package.json

```diff
@@ -28,25 +28,28 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bs-fetch": "^0.6.2",
-    "graphql": "^15.3.0",
-    "urql": "1.10.0"
+    "bs-fetch": "^0.6.2"
   },
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.9.6",
     "@glennsl/bs-jest": "^0.5.1",
+    "@reasonml-community/graphql-ppx": "^1.0.1",
     "all-contributors-cli": "^6.14.2",
     "babel-jest": "^26.0.1",
     "bs-platform": "7.3.2",
     "gentype": "^3.22.0",
+    "graphql": "^15.3.0",
     "husky": "^4.2.5",
     "lint-staged": "^10.2.2",
     "npm-run-all": "^4.1.5",
-    "reason-react": "^0.7.0"
+    "reason-react": "^0.7.0",
+    "urql": "~1.10.1"
   },
   "peerDependencies": {
     "@reasonml-community/graphql-ppx": "^1.0.1",
-    "reason-react": "^0.7.0"
+    "graphql": "^15.3.0",
+    "reason-react": "^0.7.0",
+    "urql": "~1.10.1"
```

Comment:
I think it's fine to allow minor ranges now? I get the motivation but given that all `~1.10.1` patches have been quite important and the ones following have been as well we could go to `^1.13.0` or `^1.14.0` which don't actually have any relevant changes

Reply:
So as I understand it, this `peerDependency` is really on `react-urql` rather than `@urql/core`. `react-urql` (from what I can tell by the `package.json` here: https://github.com/FormidableLabs/urql/blob/main/packages/react-urql/package.json#L3 is on `v1.10.3`. That's what trying to bind to w/ this range (which looks like it'll bring along `@urql/core@1.14.1` 🙌